WordPress/WordPressTest/Classes/Stores/StatsWidgetsStoreTests.swift

```diff
@@ -16,7 +17,7 @@ class StatsWidgetsStoreTests: CoreDataTestCase {
         sut = nil
     }
 
-    func testStatsWidgetsDataInitializedAfterSignDidFinish() {
+    func xTestStatsWidgetsDataInitializedAfterSignDidFinish() {
```

Comment:
I think making tests `private` disables them. Alternatively and maybe clearer in the test output, we could `XCTSkip` them.

```suggestion
    private func festStatsWidgetsDataInitializedAfterSignDidFinish() {
```

Reply:
Turns out, if you disable StatsWidgetsStoreTests, WordPressDotComAuthenticatorTests start crashing. I don't know why, and it took me long enough just to figure out this. I updated `StatsWidgetsStoreTests` to not rely on `BuildSettings.current` and re-enabled these tests. I also reported the issue with WordPressDotComAuthenticatorTests here:  https://github.com/wordpress-mobile/WordPress-iOS/issues/24247.